# Zusammenhang zwischen dem jährlichen Transportvolumen eines Staates und seiner Nähe zu Canada
//Henriette Sand, 564285

##### Nullhypothese: 
Die Höhe des jährlichen Transportwertes einzelner US-Staaten hängt nicht von der Nähe zu Canada ab.

##### Alternativhypothese
Die Höhe des jährlichen Transportwertes einzelner US-Staaten hängt von der Nähe zu Canada ab.

###### H0: µdis != µval

###### H1: µdis = µval 

In [11]:
# Daten laden
df <- read.csv("trade_values_by_state_and_year_CA.csv", header=TRUE, row.names="state")

## 1. Arithmetische Mittel berechnen

In [12]:
# Werte für Test: Arithmetisches Mittel des jährlichen Transportwertes über den Beobachtungszeitraum 2013-2018
avg_value <- apply(df, 1, mean)
avg_value

AL          AK          AS          AZ          AR          CA 
 6506312517  1105035295           0  3643090299  2163545692 45080080511 
         CO          CT          DE          DC          FL          GA 
 5197819302  5165429779  1755122975    91573201  8357434904 10602156970 
         HI          ID          IL          IN          IA          KS 
  223868849  2115090512 53493291981 19991220517  6972607777  3936486928 
         KY          LA          ME          MD          MA          MI 
11397132465  5109687534  3277928444  3661661797 10907043912 72248762055 
         MN          MS          MO          MT          NE          NV 
15240044504  3141272987  8382654260  4738054205  2656313427  2079987654 
         NH          NJ          NM          NY          NC          ND 
 6587853304 15105049629   414979006 31899062627 10087962981  5841526949 
         OH          OK          OR          PA          RI          SC 
34005779127  6080499704  5517395840 22443659198   975348861  6505626527 
         SD          TN          TX          UT          VT          VA 
 1032929296 13928704736 42400839165  3301640268  4289773162  5156016514 
         WA          WV          WI          WY 
22387038612  2934327537 11560835530  1352663940

## 2. Nach Entfernung zur canadischen Grenze gruppieren

1: keine/geringe Entfernung zu Canada

2: mittlere Entfernung zu Canada

3: große Entfernung zu Canada

In [13]:
# manuell erhobene Gruppierung nach Staat in alphabetischer Reihenfolge
distance <- c(3, 1, 3, 3, 3, 3, 2, 2, 2, 2, 3, 3, 3, 1, 1, 1, 2, 2, 2, 3, 1, 2, 2, 1, 1, 3, 2, 1, 2, 2, 1, 2, 3, 1, 2, 1, 1, 3, 2, 1, 2, 2, 2, 2, 3, 2, 1, 2, 1, 2, 1, 2)
avg_value <- cbind(avg_value, distance)

## 3. Nach Mittelwert des Transportwertes gruppieren

1: hoher jährlicher Transportwert in US-Dollar

2: mittlerer jährlicher Transportwert in US-Dollar

3: niedriger jährlicher Transportwert in US-Dollar

In [14]:
average <- avg_value[,1]

In [15]:
median(average)

[1] 5357607571

In [16]:
range(average)

[1]           0 72248762055

In [17]:
quantile(average)

0%         25%         50%         75%        100% 
          0  2864824010  5357607571 11438058231 72248762055

In [18]:
sort(avg_value[,1])

AS          DC          HI          NM          RI          SD 
          0    91573201   223868849   414979006   975348861  1032929296 
         AK          WY          DE          NV          ID          AR 
 1105035295  1352663940  1755122975  2079987654  2115090512  2163545692 
         NE          WV          MS          ME          UT          AZ 
 2656313427  2934327537  3141272987  3277928444  3301640268  3643090299 
         MD          KS          VT          MT          LA          VA 
 3661661797  3936486928  4289773162  4738054205  5109687534  5156016514 
         CT          CO          OR          ND          OK          SC 
 5165429779  5197819302  5517395840  5841526949  6080499704  6505626527 
         AL          NH          IA          FL          MO          NC 
 6506312517  6587853304  6972607777  8357434904  8382654260 10087962981 
         GA          MA          KY          WI          TN          NJ 
10602156970 10907043912 11397132465 11560835530 13928704736 15105049629 
         MN          IN          WA          PA          NY          OH 
15240044504 19991220517 22387038612 22443659198 31899062627 34005779127 
         TX          CA          IL          MI 
42400839165 45080080511 53493291981 72248762055

In [19]:
# Gruppierung nach Staat in alphabetischer Reihenfolge
value <- c(2, 3, 3, 2, 3, 1, 2, 2, 3, 3, 2, 1, 3, 3, 1, 1, 2, 2, 1, 2, 2, 2, 1, 1, 1, 2, 2, 2, 3, 3, 2, 1, 3, 1, 2, 2, 1, 2, 2, 1, 3, 2, 3, 1, 1, 2, 2, 2, 1, 2, 1, 3)
avg_value <- cbind(avg_value, value)
avg_value

avg_value   distance value
AL  6506312517 3        2    
AK  1105035295 1        3    
AS           0 3        3    
AZ  3643090299 3        2    
AR  2163545692 3        3    
CA 45080080511 3        1    
CO  5197819302 2        2    
CT  5165429779 2        2    
DE  1755122975 2        3    
DC    91573201 2        3    
FL  8357434904 3        2    
GA 10602156970 3        1    
HI   223868849 3        3    
ID  2115090512 1        3    
IL 53493291981 1        1    
IN 19991220517 1        1    
IA  6972607777 2        2    
KS  3936486928 2        2    
KY 11397132465 2        1    
LA  5109687534 3        2    
ME  3277928444 1        2    
MD  3661661797 2        2    
MA 10907043912 2        1    
MI 72248762055 1        1    
MN 15240044504 1        1    
MS  3141272987 3        2    
MO  8382654260 2        2    
MT  4738054205 1        2    
NE  2656313427 2        3    
NV  2079987654 2        3    
NH  6587853304 1        2    
NJ 15105049629 2        1    
NM   414979006 3        3    
NY 31899062627 1        1    
NC 10087962981 2        2    
ND  5841526949 1        2    
OH 34005779127 1        1    
OK  6080499704 3        2    
OR  5517395840 2        2    
PA 22443659198 1        1    
RI   975348861 2        3    
SC  6505626527 2        2    
SD  1032929296 2        3    
TN 13928704736 2        1    
TX 42400839165 3        1    
UT  3301640268 2        2    
VT  4289773162 1        2    
VA  5156016514 2        2    
WA 22387038612 1        1    
WV  2934327537 2        2    
WI 11560835530 1        1    
WY  1352663940 2        3

## Test 1: kleine Distanz zu Canada und hoher Transportwert

In [20]:
dis1 <- subset(avg_value, distance == 1)
val1 <- subset(avg_value, value == 1)
# Kontrolle: Vergleich der Größen der Stichprobe
nrow(dis1)
nrow(val1)

[1] 16

[1] 16

In [21]:
# Kontrolle: arithmetische Mittel der Stichproben
mean(dis1[,1])
mean(val1[,1])

[1] 19451559751

[1] 27043168846

In [52]:
# |T| ermitteln
t.test(dis1[,1], val1[,1], var.equal = TRUE)


	Two Sample t-test

data:  dis1[, 1] and val1[, 1]
t = -1.1191, df = 30, p-value = 0.272
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -21445979461   6262761271
sample estimates:
  mean of x   mean of y 
19451559751 27043168846 


In [54]:
# T* ermitteln
qt(p = 0.05/2, df = 30, lower.tail = FALSE)

[1] 2.042272

###### Ergebnis

|T| = -1.1191

T* = 2.04227245630124

|T| < T* Nullhypothese annehmen, Alternativhypothese verwerfen

## Test 2: mittlere Distanz zu Canada und mittlerer Transportwert

In [24]:
dis2 <- subset(avg_value, distance == 2)
val2 <- subset(avg_value, value == 2)
# Kontrolle: Vergleich der Größen der Stichproben
nrow(dis2)
nrow(val2)

[1] 23

[1] 23

In [25]:
# Kontrolle: arithmetische Mittel der Stichproben
mean(dis2[,1])
mean(val2[,1])

[1] 5569630418

[1] 5408394066

In [49]:
# |T| ermitteln
t.test(dis2[,1], val2[,1], var.equal = TRUE)


	Two Sample t-test

data:  dis2[, 1] and val2[, 1]
t = 0.16644, df = 44, p-value = 0.8686
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -1791123654  2113596357
sample estimates:
 mean of x  mean of y 
5569630418 5408394066 


In [50]:
# T* ermitteln
qt(p = 0.05/2, df = 44, lower.tail = FALSE)

[1] 2.015368

###### Ergebnis

|T| = 0,16644

T* = 2.01536757444376

|T| < T* Nullhypothese annehmen, Alternativhypothese verwerfen

## Test 3: große Distanz zu Canada und niedriger Transportwert

In [28]:
dis3 <- subset(avg_value, distance == 3)
val3 <- subset(avg_value, value == 3)
# Kontrolle: Vergleich der Größen der Stichproben
nrow(dis3)
nrow(val3)

[1] 13

[1] 13

In [44]:
# Kontrolle: arithmetische Mittel der Stichproben
mean(dis3[,1])
mean(val3[,1])

[1] 10286443703

[1] 1228189131

In [55]:
# |T| ermitteln
t.test(dis3[,1], val3[,1], var.equal = TRUE)


	Two Sample t-test

data:  dis3[, 1] and val3[, 1]
t = 2.146, df = 24, p-value = 0.0422
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
   346394907 17770114236
sample estimates:
  mean of x   mean of y 
10286443703  1228189131 


In [56]:
# T* ermitteln
qt(p = 0.05, df = 24, lower.tail = FALSE)

[1] 1.710882

## Ergebnis

|T| = 2.146

T* = 1.71088207990943

|T| > T* Nullhypothese verwerfen, Alternativhypothese annehmen